app/Http/Controllers/API/VoucherController.php

```diff
@@ -83,20 +86,58 @@ public function transition(Request $request)
                 $voucher->applyTransition($transition);
                 // Success for this one.
                 $success_codes[] = $voucher->code;
+                // If this is a confirm transition - add to a list
+                // for sending to ARC admin. This is a request for payment.
+                if ($transition === 'confirm') {
+                    $vouchers_for_payment[] = $voucher;
+                }
             } else {
                 // no! add it to a list of failures.
                 // Fail for this one.
                 $fail_codes[] = $voucher->code;
             }
         }
 
+        // If there are any confirmed ones... trigger the email
+        // event paymentRequestedEmail.
+        if (sizeof($vouchers_for_payment) > 0) {
+            // Todo If the email fails for some reason, the User will not be aware.
+            // The Vouchers will be transitioned but the ARC admin will not know.
+            $this->emailVoucherPaymentRequest($trader, $vouchers_for_payment);
```

Comment:
This comment is correct - email is not a garuanteed medium. However, in the distant future we'd have an actiivities function in the portal - this card is marked as a stopgap until then.